pkg/sysprom/initialize_test.go

```diff
@@ -0,0 +1,54 @@
+package sysprom
+
+import (
+	"flag"
+	"reflect"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+var mode = flag.String("mode", "standalone", "kubernetes configuration mode")
+
+func TestNewConfig(t *testing.T) {
+	viper.Set("mode", mode)
```

Comment:
You can move this to TestSetup method, instead of doing it over and over in all tests.

Reply:
Moved viper.Set() to init function